src/qutip_qip/qiskit/provider.py

```diff
@@ -16,7 +18,10 @@ def __init__(self):
         super().__init__()
 
         self.name = "qutip_provider"
```

Comment:
The coverage shows that the `provider.py` is not much tested.
https://coveralls.io/builds/52369228/source?filename=src%2Fqutip_qip%2Fqiskit%2Fprovider.py#L21

Is there a way to add a test for it?